exe/kubernetes-deploy

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
+require 'kubernetes-deploy'
+
+options = {
+  namespace: ARGV[0],
+  context: ARGV[1],
+  environment: ENV['ENVIRONMENT'],
+  current_sha: ENV['REVISION'],
+  kubeconfig_path: ENV["KUBECONFIG"],
+}
```

Comment:
We should consider using proper arguments down the road with a good `--help` prompt. Not needed now, but when we add more arguments that'd be extremely helpful.